<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="pages">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="yuhunpage">
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <attribute name="title">
      <string>御魂</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>256</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QCheckBox" name="yuhun_isTeamMode">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>组队模式</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QComboBox" name="yuhun_chooseFloor">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="currentText">
          <string>请选择副本类型</string>
         </property>
         <item>
          <property name="text">
           <string>请选择副本类型</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1~10层</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>悲鸣</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>贪</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>嗔</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>痴</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>日轮之城 1层</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>日轮之城 2层</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>日轮之城 3层</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="yuhun_editPersonCount"/>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="yuhun_isAddOtherOperations">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>添加随机操作(推荐)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="yuhun_editPropCount"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="yuhun_label_propcount">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>设置樱饼数量:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="yuhun_label_personcount">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>人数(上限3):</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="yuhun_label_challengetimes">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>设置挑战次数:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="yuhun_editChallengetimes"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>119</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="yuhun_button_start">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yuhun_button_pause">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yuhun_button_stop">
         <property name="font">
          <font>
           <family>宋体</family>
          </font>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="juexingpage">
     <attribute name="title">
      <string>觉醒</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>119</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="juexing_button_start">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="juexing_button_pause">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="juexing_button_stop">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>243</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="1">
        <widget class="QLineEdit" name="juexing_editPersonCount">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="juexing_editPropCount">
         <property name="minimumSize">
          <size>
           <width>58</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>58</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="juexing_label_personcount">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>人数(上限3):</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="juexing_isTeamMode">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>组队模式</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="juexing_label_propcount">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置樱饼数量:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="yulingpage">
     <attribute name="title">
      <string>御灵</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>256</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QCheckBox" name="yuling_isAddOtherOperations">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>添加随机操作(推荐)</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="yuling_editChallengetimes"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="yuling_label_challengetimes">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>设置挑战次数:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>119</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="yuling_button_start">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yuling_button_pause">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yuling_button_stop">
         <property name="font">
          <font>
           <family>宋体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>停止</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="help">
     <attribute name="title">
      <string>帮助</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>500</width>
        <height>270</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;1.使用本程序时，请将界面调至挑战页面(组队页面)。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;2.本程序不提供准备战斗功能，使用时请确保锁定阵容。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;3.本程序运行过程中，阴阳师客户端会被锁定，期间无法拖动其他窗口。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;4.按快捷键ctrl+Q可快速暂停本程序，以便操作其他窗口。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTextBrowser" name="showInfoArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>500</width>
      <height>130</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;color:rgb(127,255,0)</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;输出信息:&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>412</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="resolution_label">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>   分辨率:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="resolution1">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1440*849(默认)</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="resolution2">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>961*579(推荐)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>showInfoArea</zorder>
   <zorder>pages</zorder>
   <zorder>horizontalLayoutWidget_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
